corrige les liens vers les images et style pour les epub

artragis opened the pull request:
- générer un epub avec des images et des blocs d'information
- s'assurer que tout s'affiche avec le bon style.

Commit:
corrige les liens vers les images et style pour les epub

Commit:
retire une erreur lorsque la gallerie n'est pas bonne

pierre-24 commented:
Rapport de QA: ok pour les images, moins ok pour les blocs : 

![screenshot from 2018-08-03 18-08-49](https://user-images.githubusercontent.com/7889675/43653462-673e5d9c-9748-11e8-867c-336a76627d59.png)

artragis commented:
As-tu l'ebook généré ? Je regarderai dimanche ou lundi ce qui ne va pas.

pierre-24 commented:
J'ai du faire un *wipe*, mais je peux re-générer ça ;)

pierre-24 commented:
En voilà un : https://documents.unamur.be/index.php/s/kNbfbd5E74SX39p :)

artragis commented:
bon, j'ai compris pourquoi et... bon sang que je m'en veux.

Je fix ça demain.

Commit:
Fix le nom des fichiers de style & les identifiants

en XML on ne peut pas avoir d'@ dans les id, or les images de nos asset ont un @ dans le path et je me sers du path
pour générer l'id.
J'ai donc remplacé le @ dans l'id par un simple -.

artragis commented:
@pierre-24 ça devrait être bon.

pierre-24 commented:
Oui :)

![screenshot from 2018-08-05 13-47-48](https://user-images.githubusercontent.com/7889675/43685604-40ed8a22-98b6-11e8-8c33-218bf2e26558.png)
